src/components/molecules/FormFields/WarnCheckbox.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { ReactNode, FormEvent, ReactElement, ChangeEvent } from 'react'
```

Comment:
why is this a new component? It's a simple checkbox so why create a dedicated component for one specific checkbox instead of using our established `InputElement`? Same goes for the above styles which are an exact copy of the checkbox styles from that `InputElement` needlessly duplicating code.

More importantly, why a user prompt for acknowledgement which is not present in any other pricing method, making this highly inconsistent?  To me there should be some representation of free (either big "Free" or a "0 OCEAN") and the datatoken still being represented, just like on fixed pricing.

Reply:
Got it, I will try to update this to a big "Free"